Makefile

```diff
@@ -20,7 +20,7 @@ all: server
 
 .PRECIOUS: ${PROTO_DIR}/%_openapi.yaml ${PROTO_DIR}/%.proto
 
-BUILD_PARAM=-tags=release -ldflags "-X 'main.Version=$(VERSION)' -X 'main.BuildHash=$(GIT_HASH)'" $(shell printenv BUILD_PARAM)
+BUILD_PARAM=-tags=release -ldflags "-X 'github.com/tigrisdata/tigris/util.Version=$(VERSION)' -X 'github.com/tigrisdata/tigris/util.BuildHash=$(GIT_HASH)'" $(shell printenv BUILD_PARAM)
```

Comment:
I've removed GIT_HASH in client repo, I think it's redundant. There is enough information in VERSION itself.
It'll be symbolic name if tagged and symbolic+hash if not tagged.

Reply:
removed `BuildHash`